app/scripts/controllers/bridge-status/constants.ts

```diff
@@ -1,10 +1,17 @@
-import { BridgeStatusControllerState } from '../../../../shared/types/bridge-status';
+import type {
+  BridgeStatusControllerState,
+  BridgeStatusState,
+} from '../../../../shared/types/bridge-status';
 
 export const REFRESH_INTERVAL_MS = 10 * 1000;
 
 export const BRIDGE_STATUS_CONTROLLER_NAME = 'BridgeStatusController';
 
+export const DEFAULT_BRIDGE_STATUS_STATE: BridgeStatusState = {
+  txHistory: {},
+};
+
 export const DEFAULT_BRIDGE_STATUS_CONTROLLER_STATE: BridgeStatusControllerState =
   {
-    txHistory: {},
+    bridgeStatusState: DEFAULT_BRIDGE_STATUS_STATE,
   };
```

Comment:
- Rename `DEFAULT_BRIDGE_STATUS_CONTROLLER_STATE` to `DEFAULT_BRIDGE_STATUS_STATE`
- Define new `DEFAULT_BRIDGE_STATUS_CONTROLLER_STATE`